dbgl.spec

```diff
@@ -35,52 +46,83 @@ the product is somewhat usable as it is. Please bear in mind that
 the interface is still quite rough around the edges.
 
 %prep
-%setup -c
+%setup -qc
+# remove bundles
+# swt
+rm -rf ./src/dist/linuxshared/lib
+rm -rf ./src/dist/*/DOSBox-0.74
+rm -r ./src/dist/shared/lib/hsqldb.jar
+rm -r ./src/dist/shared/lib/commons-lang3-3.4.jar
+rm -r ./src/dist/shared/lib/commons-io-2.4.jar
+# not easy unbundle jersey-2.13.jar fedora have jersey-2.18 and 2.23 seems that
+# haven't org.glassfish.jersey.core.jersey-server
+#rm -r ./src/dist/shared/lib/jersey-2.13.jar
+
 
 %build
-ant
-tar -xf ./dist/dbgl080_generic.tar.gz
+mkdir -p lib
+build-jar-repository -p lib commons-lang3 hsqldb swt \
+    commons-io
+#    glassfish-jax-rs-api jersey glassfish-hk2-utils \
+#    glassfish-hk2-configuration glassfish-hk2/hk2-metadata-generator \
+#    glassfish-hk2-hk2
 
-%install
+ant distlinux
 
-# jars
+%install
 install -dm 755 %{buildroot}%{_javadir}/%{name}/
-copy_files="captures db  dbgl.jar  dosroot export lib profiles templates xsl"
-for files in $copy_files
-do 
-cp -af $files %{buildroot}/%{_javadir}/%{name}/
-done
+%ifarch x86_64
+    tar xvf dist/dbgl080_64bit.tar.gz -C %{buildroot}/%{_javadir}/%{name}/
+%else
+    %ifarch i686
```

Comment:
yes , %{ix86} is more correct  and also I will remove / between %{buildroot} and others.